src/lib/format.pl

```diff
@@ -699,6 +699,13 @@
 
 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
+%% listing(+PI)
+%
+%  Given a predicate indicator PI=Identifier/Arity, `listing(PI)` will
```

Comment:
Why future tense?

Reply:
I think I was thinking in future tense because there's impure IO and that's how my brain works.